vendor/autoload.php

```diff
@@ -0,0 +1,7 @@
+<?php
```

Comment:
the vendor folder should go in the `.gitignore`, please remove the entire folder from git